### PR TITLE
LINK-1646 | Bump HDS to version 3.1.0 and update header

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
- Bump HDS to version 3.1.0
- Change HDS Navigation component to Header component

## Closes
[LINK-1646](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1646)

## Screenshot
Screenshot 2023-10-17 at 14 04 37

[LINK-1646]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by arpargo** [comment]
Looks good, seems to mostly work.
Two things I noticed:
a) The close text in right corner of the screenshot is clearly missing translations (was there on testing aswell)
b) For some reason, when on the base url (localhost:3001) the finnish language doesn't work locally? Not sure if this is a feature or not but thought I should mention it. Translations work properly on other 404 pages like '/asd'.

**Comment by jorilindell** [comment]
> Looks good, seems to mostly work. Two things I noticed: a) The close text in right corner of the screenshot is clearly missing translations (was there on testing aswell) b) For some reason, when on the base url (localhost:3001) the finnish language doesn't work locally? Not sure if this is a feature or not but thought I should mention it. Translations work properly on other 404 pages like '/asd'.

- Added translations for the close text
- Added prefix to default locale to avoid problems with automatic language detection. See https://nextjs.org/docs/pages/building-your-application/routing/internationalization#prefixing-the-default-locale

File: public/locales/sv/common.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
   "appName": "Linked Registrations",
+  "ariaLogo": "Service logo",
```
Comment (harriris-vincit) [inline public/locales/sv/common.json:3]: I think "Tjänstens logo" is the proper Swedish translation

Comment (jorilindell) [reply public/locales/sv/common.json]: Changed

**Review by harriris-vincit — APPROVED** [review]
Looks good

**Pull request merged**